README.md

```diff
@@ -101,7 +101,12 @@ as a compilation tool on top of Go:
 
     $ go get -u github.com/mitchellh/gox
 
-Next, clone this repository into `$GOPATH/src/github.com/mitchellh/packer`.
+Next, clone this repository into `$GOPATH/src/github.com/mitchellh/packer`:
```

Comment:
Noted.  I'll try to find time to test that out on the vagrant box and get that suggested change into the readme.  Thanks again.